src/app/app-routing.module.ts

```diff
@@ -62,6 +63,7 @@ const appRoutes: Routes = [
   { path: 'wizard-step-directives', component: WizardStepDirectivesComponent },
   { path: 'reset-wizard', component: ResetWizardComponent },
   { path: 'reversed-navigation-bar', component: ReversedNavigationBarComponent },
+  { path: 'fix-for-ngfor', component: FixForNgforComponent },
```

Comment:
I think the path and the component should be called differently, because we're not trying to provide a fix here but an general example how to use `ngFor` with the `wizard-step` component.